tokio/src/runtime/handle.rs

```diff
@@ -28,23 +31,144 @@ pub(crate) struct Handle {
     pub(super) blocking_spawner: blocking::Spawner,
 }
 
+/// Runtime context guard.
+///
+/// Returned by [`Runtime::enter`] and [^Handle::enter^], the context guard exits
```

Comment:
are the ^s a typo, or does that mean something to RustDoc? to me, it _looks_ like this should be
```suggestion
/// Returned by [`Runtime::enter`] and [`Handle::enter`], the context guard exits
```

Reply:
Definitely a typo